en/0x10-V1-Inventory.md

```diff
@@ -17,13 +17,13 @@ transparency within the organization, promotes software standardization and reus
 
 | # | Description | L1 | L2 | L3 |
 | :---: | :--- | :---: | :---: | :---: |
-| **1.1** | All components and their versions are known at completion of a build | ✓ | ✓ | ✓ |
-| **1.2** | Package managers are used to manage all third-party binary components | ✓ | ✓ | ✓ |
+| **1.1** | All constituent components and their versions are known at completion of a build | ✓ | ✓ | ✓ |
+| **1.2** | Package managers are used to manage all third-party components | ✓ | ✓ | ✓ |
 | **1.3** | An accurate inventory of all third-party components is available in a machine-readable format | ✓ | ✓ | ✓ |
 | **1.4** | Software bill-of-materials are generated for publicly or commercially available applications | ✓ | ✓ | ✓ |
-| **1.5** | Software bill-of-materials are required for new procurements | | ✓ | ✓ |
-| **1.6** | Software bill-of-materials are continuously maintained for all systems | | | ✓ |
-| **1.7** | Uniquely identify all components in a consistent, machine-readable format | ✓ | ✓ | ✓ |
+| **1.5** | Software bill-of-materials are required for new components | | ✓ | ✓ |
+| **1.6** | Software bill-of-materials are maintained to be current for all systems | | | ✓ |
```

Comment:
Keep original text but change slightly to:
"Software bill-of-materials are continuously maintained and current for all systems"

Reply:
I think you then need to define "continuously". I have updated.